components/aria2c/Dockerfile

```diff
@@ -0,0 +1,40 @@
+# Pulling ubuntu bionic as base image
+FROM ubuntu:18.04
```

Comment:
For aria2c, we can use alpine:edge as I tried an existing aria2c docker file and it ran smoothly
Have a look on this @kmehant https://github.com/XUJINKAI/aria2-with-webui/blob/master/Dockerfile

Reply:
Sure will shift it to alpine :)

> For aria2c, we can use alpine:edge as I tried an existing aria2c docker file and it ran smoothly
> Have a look on this @kmehant https://github.com/XUJINKAI/aria2-with-webui/blob/master/Dockerfile

Reply:
> For aria2c, we can use alpine:edge as I tried an existing aria2c docker file and it ran smoothly
> Have a look on this @kmehant https://github.com/XUJINKAI/aria2-with-webui/blob/master/Dockerfile
Done 👍